docs/docs/modules/indexes/document_loaders/examples/web_loaders/open_ai_whisper_audio.mdx

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think this one should be a file loader since it only works with local files?

And actually we probably should have split the other one into two different loaders :/